packages/tooltip/Tooltip.test.tsx

```diff
@@ -1,17 +1,49 @@
-import { Tooltip } from './Tooltip';
-
+import { Tooltip } from './index';
 import { h, mount } from 'bore';
-import { emit } from 'skatejs';
 
 describe( Tooltip.is, () => {
 
   it( 'should render', () => {
 
-    console.warn('Missing tests for Tooltip!');
-
     return mount(
       <Tooltip label="hello"/>
-    ).wait();
+    ).wait((element) => {
+      expect( element.has('.c-tooltip') ).to.be.true;
```

Comment:
Great :) we have been discussing this yesterday with @elmariofredo  & @davidrus so we can take the opportunity and agree what should be the preferred version.